scripts/binary_release.sh

```diff
@@ -40,22 +45,32 @@ function upload_binary {
   then
     tar_binary $1
     tar_file=$1.tar.gz
-    aws s3 cp --acl public-read target/release/$1 s3://build.nearprotocol.com/nearcore/${os}/${BRANCH}/$1
-    aws s3 cp --acl public-read target/release/$1 s3://build.nearprotocol.com/nearcore/${os}/${BRANCH}/${COMMIT}/$1
-    aws s3 cp --acl public-read target/release/$1 s3://build.nearprotocol.com/nearcore/${os}/${BRANCH}/${COMMIT}/stable/$1
 
-    aws s3 cp --acl public-read target/release/$1 s3://build.nearprotocol.com/nearcore/${os_and_arch}/${BRANCH}/$1
-    aws s3 cp --acl public-read target/release/$1 s3://build.nearprotocol.com/nearcore/${os_and_arch}/${BRANCH}/${COMMIT}/$1
-    aws s3 cp --acl public-read target/release/$1 s3://build.nearprotocol.com/nearcore/${os_and_arch}/${BRANCH}/${COMMIT}/stable/$1
+    upload_targets=()
+    if [ -n "${BRANCH}" ]; then
+      upload_targets+=("${BRANCH}")
+    fi
+    if [ -n "${RELEASE_TAG}" ] && { [ -z "${BRANCH}" ] || [ "${BRANCH}" != "${RELEASE_TAG}" ]; }; then
```

Comment:
If release tag is non-empty and doesn't equal branch it would cover the case of branch being empty or do I misunderstand something?

```suggestion
    if [ -n "${RELEASE_TAG}" ] && [ "${BRANCH}" != "${RELEASE_TAG}" ]; then
```